src/lwc/geFormSection/geFormSection.js

```diff
@@ -263,11 +272,58 @@ export default class GeFormSection extends LightningElement {
                 // Currently only picklists need their selected record's RecordType Id,
                 // since they use it to update their available options
                 if (field.isPicklist) {
-                    if (field.objectMappingDevName === objectMappingDevName) {
+                    if (field.targetObjectMappingDevName === objectMappingDevName) {
                         field.recordTypeId = recordTypeId;
                     }
                 }
             });
     }
 
+}
+
+class GeFormElement {
```

Comment:
@psadie-sfdo this one was part of the merge from @bdvorachek's branch. This component wasn't edited as part of this PR. I do agree this could be a good candidate for a separate class though.